From 5b52ee6c6efab68111d128d45f386ac21eaf84f6 Mon Sep 17 00:00:00 2001
From: Brian Dolbec <dolsen@gentoo.org>
Date: Sun, 10 Jul 2022 13:41:36 -0700
Subject: [PATCH] Revert "setup.py: migrate to setuptools"

This reverts commit bbbde97b5e625a49a1a66e307931548cb33f260b.
setuptools only installs data files to the python pkg directory
---
 setup.py | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/setup.py b/setup.py
index 36995de..23e9b36 100755
--- a/setup.py
+++ b/setup.py
@@ -3,8 +3,8 @@
 import re
 import sys
 import subprocess
-
-from setuptools import setup, Command
+from distutils import core
+from distutils.cmd import Command
 from glob import glob
 
 import os
@@ -67,7 +67,7 @@ manpages = [
 ]
 
 
-class set_version(Command):
+class set_version(core.Command):
     """Set python __version__ and bash VERSION to our __version__."""
 
     description = "hardcode scripts' version using VERSION from environment"
@@ -130,7 +130,7 @@ test_data = {
     ]
 }
 
-setup(
+core.setup(
     name="gentoolkit",
     version=__version__,
     description="Set of tools that work with and enhance portage.",
--
libgit2 1.4.3

